holoviews/operation/datashader.py

```diff
@@ -374,9 +373,10 @@ def _process(self, element, key=None):
                 raise ValueError("Aggregation column %s not found on %s element. "
                                  "Ensure the aggregator references an existing "
                                  "dimension." % (column,element))
+            name = column
             if isinstance(agg_fn, ds.count_cat):
-                name = '%s Count' % agg_fn.column
-            vdims = [dims[0](column)]
+                name = '%s Count' % column
```

Comment:
``name = '%s Count' % column if isinstance(agg_fn, ds.count_cat) else column``?

Personally, I don't like it when I see simple variable reassignments e.g ``x=y`` used in this way. Up to you, I won't hold the merge up because of this.

Reply:
There is more to the if/else block than just this single line.

Reply:
Suppose I see what you're getting at.